pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -143,6 +143,27 @@ func (c *cephStatusChecker) checkStatus() {
 			logger.Errorf("failed to delete pod on not ready nodes. %v", err)
 		}
 	}
+
+	c.configureHealthSettings(status)
+}
+
+func (c *cephStatusChecker) configureHealthSettings(status cephclient.CephStatus) {
+	// loop through the health codes and log what we find
+	for healthCode, check := range status.Health.Checks {
+		logger.Debugf("Health: %q, code: %q, message: %q", check.Severity, healthCode, check.Summary.Message)
+	}
+
+	// disable the insecure global id if there are no old clients
+	if _, ok := status.Health.Checks["AUTH_INSECURE_GLOBAL_ID_RECLAIM_ALLOWED"]; ok {
+		if _, ok := status.Health.Checks["AUTH_INSECURE_GLOBAL_ID_RECLAIM"]; !ok {
+			logger.Info("disabling the insecure global ID as no legacy clients are currently connected")
```

Comment:
Might want to check that `auth_expose_insecure_global_id_reclaim` is set to true on all monitors before making decisions based on lack of `AUTH_INSECURE_GLOBAL_ID_RECLAIM` -- otherwise it wouldn't be raised in most cases.

OTOH `auth_expose_insecure_global_id_reclaim` is set to true by default and basically no one should ever touch it.  Up to you...